lib/volcanic/authenticator/v1/privilege.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Volcanic::Authenticator
+  module V1
+    # Privilege api
+    class Privilege < Common
+      class << self
+        def path
+          'api/v1/subject'
```

Comment:
this path is not for privileges - it is about  subjects.

The only time it makes sense with privileges is when trying to find the privileges for a single subject.

All other interactions would be with the privileges API

Reply:
Updated to be a `Subject` class, rightly so too 👍